sdk_extensions/aws_v1_support/src/main/java/io/opentelemetry/sdk/extensions/trace/aws/resource/BeanstalkResource.java

```diff
@@ -95,6 +95,8 @@ public Attributes getAttributes() {
       return Attributes.empty();
     }
 
+    attrBuilders.setAttribute(ResourceAttributes.CLOUD_PROVIDER, "aws");
```

Comment:
Should we use a constant for the name? I do not expect AWS to change its name soon :stuck_out_tongue: but it might saves us some headache if we'll change the capitalization.

Reply:
Makes sense, thanks